weave-js/src/components/PagePanelComponents/Home/Browse3/NotFoundPanel.tsx

```diff
@@ -0,0 +1,19 @@
+import {ErrorPanel} from '@wandb/weave/components/ErrorPanel';
```

Comment:
I noticed while testing that it is pretty lame when a not found error crashes the whole page. This is a common component that can be used to render not found data